content/en/continuous_integration/intelligent_test_runner/java.md

```diff
@@ -11,8 +10,6 @@ further_reading:
       text: "Troubleshooting CI"
 ---
 
-{{< callout url="#" btn_hidden="true" >}}Intelligent Test Runner for Java in beta.{{< /callout >}} 
-
```

Comment:
This one we still want to keep, right @nikita-tkachenko-datadog ?

Reply:
Yep, Java is in public beta for now